app/controllers/api/v1/admin/user_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+    module V1
+        module Admin
+            class UserController < Api::V1::Admin::AdminApiController
+   
+                def index
+                    users = User.all
+                    render json:users, status: :ok
+                end
+                
+                def show
+                    user = User.find(params[:id])
+                    render json:user, status: :ok
```

Comment:
Still have a lot of this renders

Reply:
esto es porque es el user_controller y no el users_controller , ya borre el anterior